### ISSUE TITLE
unable to access user_uploads images

### ISSUE THREAD

**Issue by lgsandy** [zulip/zulip-js#107]
I have implemented zulip chat in my web app with zulip-js. I have uploaded images from chat with zulip api but while accessing in web with url i am getting 403 error.
<img width="957" alt="Zulip error" src="https://github.com/zulip/zulip-js/assets/32234544/f96d6224-954b-439a-b986-bf0de9e393d6">
<img width="587" alt="zulip file" src="https://github.com/zulip/zulip-js/assets/32234544/e808c5f0-eb73-4dae-8e12-b247acf32579">

**Comment by karlstolley** [comment]
Hi @lgsandy -- when you say "accessing in the web," do you mean you're trying to get your images outside of Zulip by navigating directly to one of those URLs?

**Comment by lgsandy** [comment]
I am fetching chat from zulip wth zulip-js and in that i am getting image url  that url is showing 403.

**Comment by lgsandy** [comment]
Yes in local. After logged in in zulip app in same browser that url is working.But through zulip-js lib it is not accessing.

**Comment by karlstolley** [comment]
Hmmmmm. @alexmv, might you have an idea of what's going on here?
